packages/server/src/controllers/auth.ts

```diff
@@ -0,0 +1,30 @@
+import { getAccountByEmail } from "../db/accounts";
+import { Context } from "koa";
+import { createToken } from "../utils";
+
+export const login = async (ctx: Context) => {
+  const { email } = ctx.request.body;
+  const member = await getAccountByEmail(email);
+  console.log(member);
```

Comment:
Pas contre l'idée de logger des infos dans la console, mais la je ne vois pas à quoi ça servirai. Vaut mieu logger un truc du genre `${email} is trying to login` ou `${member.firstname} ${member.lastname} successfully logged in on ${new Date()}`

Reply:
Non là c'était juste pour vérifier ce que je récupérais :)) 
Je mets en place un système de logging/alerting cette semaine :))